app/code/Magento/Inventory/Model/SourceStockLinkManagement.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Model;
+
+use Magento\Framework\Api\SearchCriteria\CollectionProcessorInterface;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Inventory\Model\ResourceModel\SourceStockLink\CollectionFactory as StockLinkCollectionFactory;
+use Magento\InventoryApi\Api\Data\StockInterface;
+use Magento\InventoryApi\Api\SourceStockLinkManagementInterface;
+
+/**
+ * Class to get assign sources by stock.
+ */
+class SourceStockLinkManagement implements SourceStockLinkManagementInterface
```

Comment:
for now we only have SourceStockLinkManagementInterface
https://github.com/magento-engcom/magento2/blob/msi-stock-ui-aggregation/app/code/Magento/InventoryApi/Api/SourceStockLinkManagementInterface.php

which provides a possibility to get  assigned sources to specified stock.
But we also need another operation - to assign sources to specified stock

`assignSourcesToStock(array $sourceIds, $stockId);`

I propose to make next refactoring.

Current SourceStockLinkManagementInterface rename to 

```php
interface GetAssignedSourcesByStockIdCommand
{
    public function execute($stockId) 
    { 
      //...
    }
}

//Introduce new Interface

interface AssignSourcesToStockInterface
{
    public function execute(array $sourceIds, $stockId) 
    { 
      //...
    }
}


```